backend/appwrite-functions/set-wallet/main.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+from appwrite.client import Client
+from appwrite.services.database import Database
+
+# provided by default
+USER_ID = os.environ.get('APPWRITE_FUNCTION_USER_ID')
+PAYLOAD = os.environ.get('APPWRITE_FUNCTION_DATA')
+APPWRITE_PROJECT = os.environ.get('APPWRITE_FUNCTION_PROJECT_ID')
+
+# provided by cloud function env vars
+APPWRITE_ENDPOINT = os.environ.get('APPWRITE_ENDPOINT')
+APPWRITE_API_KEY = os.environ.get('APPWRITE_API_KEY')
+WALLETS_COLLECTION_ID = os.environ.get('WALLETS_COLLECTION_ID')
+
+client = Client()
+database = Database(client)
+
+(client
```

Comment:
Are these settings related to the database query? Would it make a difference if these setters are written before the `Database` constructor?

Reply:
Don't know which setters you mean.

Reply:
Okay, I mean: would it make a difference to write this `client` statement in parentheses with its `client.setX` calls before the database?